lib/placeholders/card/readMe.md

```diff
@@ -0,0 +1,35 @@
+## Navbar
+
+The card placeholder component.
+
+### Properties
+
+| Name         | Type     | Default  | Description                                 |
+| :----------- | :------- | :------- | :------------------------------------------ |
+| `imageShape` | `String` | `circle` | Set the shape of the box in card component. |
+
+### Theme
+
+| Name         | Description                      |
+| :----------- | :------------------------------- |
+| `cardStyles` | Class used for the root element. |
+
+### Usage
+
+```
+
+const Demo = ({ theme, imageShape }) => (
+  <div className={classNames(theme.styles, theme.cardStyles)}>
+    <div className={theme.imageWrapper}>
+      <div className={theme.gutterSpaceHorizontal}>
+        <BoxPlaceholder shape={imageShape} />
+      </div>
+
+      <LinePlaceholder lineWidth="100" />
+    </div>
+
+    <LinePlaceholder lineWidth="75" />
+    <LinePlaceholder lineWidth="50" />
+  </div>
+);
```

Comment:
As mentioned earlier, Please Update Usage here also.

You can have the above usage too, to show how the card placeholder can be constructed using `Box` and `LinePlaceholders` we have.


```jsx
class Demo extends React.Component {
    state = {
       loading: false,
    }
    render() {
       const {loading} = this.state;
       return (
            <div>
                { loading ? <CardPlaceholder /> : <div>User Card</div>
             </div>
        )
    }
}```

Reply:
1309bc5f877031f4e7719566dc665b2940b86281